src/shared/api/generated/GenomeNexusAPIInternal-docs.json

```diff
@@ -12,7 +12,7 @@
       "url": "https://github.com/cBioPortal/genome-nexus/blob/master/LICENSE"
     }
   },
-  "basePath": "/",
+  "basePath": "/beta",
```

Comment:
@inodb is this right?  shouldn't this file be env agnostic?

Reply:
@alisman Probably not right, good point. I think the API generator tries to guess the `basePath` depending on the provided url, and that url depends on the genome nexus env parameter.

Reply:
I don't think this parameter ends up in the generated client so it might not be a big deal. Just API sync error will prolly be raised on master when  generating client from genomenexus.org . Maybe we should strip out basepath info from the docs (add grep -v in the updateAPI script)